src/renderer/features/assets/AssetsPortfolio/ui/TokenBalanceList.tsx

```diff
@@ -0,0 +1,114 @@
+                  </FootnoteText>
+                  <ChainIcon src={chains[asset.chains[0].chainId].icon} name={asset.chains[0].name} size={18} />
+                  <ChainIcon
+                    className="mx-[-8px]"
+                    src={chains[asset.chains[1].chainId].icon}
+                    name={asset.chains[1].name}
+                    size={18}
+                  />
+                  {asset.chains.length > 2 && (
+                    <div className="b-r-2 w-6 rounded flex items-center justify-center bg-token-background p-0.5">
+                      <HelpText className="text-white">+1</HelpText>
+                    </div>
+                  )}
+                  {asset.totalBalance?.verified && (
+                    <div className="flex items-center gap-x-2 text-text-warning ml-2.5">
+                      <Tooltip content={t('balances.verificationTooltip')} pointer="up">
+                        <Icon name="warn" className="cursor-pointer text-inherit" size={14} />
+                      </Tooltip>
+                    </div>
+                  )}
+                </div>
+              </div>
+            </div>
+            <TokenPrice
+              assetId={asset.priceId}
+              wrapperClassName="flex-col gap-0.5 items-end px-2 w-[100px]"
+              className="text-text-primar text-right"
+            />
+            <div className="flex flex-col items-end w-[100px]">
+              {asset.totalBalance?.free ? (
+                <>
+                  <AssetBalance value={totalAmount(asset.totalBalance)} asset={asset} showSymbol={false} />
+                  <AssetFiatBalance amount={totalAmount(asset.totalBalance)} asset={asset} />
+                </>
+              ) : (
+                <div className="flex flex-col gap-y-1 items-end">
+                  <Shimmering width={82} height={20} />
+                  {fiatFlag && <Shimmering width={56} height={18} />}
+                </div>
+              )}
+            </div>
+
+            <div className="flex gap-x-2 ml-3">
+              <CheckPermission operationType={OperationType.TRANSFER} wallet={activeWallet}>
+                <Link to={createLink(Paths.TRANSFER_ASSET, {})} onClick={(e) => e.stopPropagation()}>
+                  <Icon name="sendArrow" size={20} />
+                </Link>
+              </CheckPermission>
+              <CheckPermission operationType={OperationType.RECEIVE} wallet={activeWallet}>
+                <Link to={createLink(Paths.RECEIVE_ASSET, {})} onClick={(e) => e.stopPropagation()}>
+                  <Icon name="receiveArrow" size={20} />
+                </Link>
+              </CheckPermission>
+            </div>
+          </div>
+        </Accordion.Button>
+
+        <Accordion.Content className="mt-1">
+          <ul className="flex flex-col gap-y-1.5 pl-6">
+            {asset.chains.map((chain) => (
+              <NetworkCard key={chain.chainId + chain.assetId} chain={chain} asset={asset} />
```

Comment:
Maybe use template literals here? Concat string with + may provide some issues in future